source/pay-product-page/stylesheets/modules/_cookie-settings.scss

```diff
@@ -0,0 +1,32 @@
+.js-enabled {
+    .cookie-settings__no-js{
+        display: none;
+    }
+}
+
+.pay-cookie-settings-alert {
+    @include govuk-font(19);
+    display: block;
+    padding: govuk-spacing(3);
+    margin: govuk-spacing(3) 0 govuk-spacing(6) 0;
+    text-align: left;
+    position: relative;
+    clear: both;
+    border-style: solid;
+    border-width: 5px;
+}
+
+.pay-cookie-settings-alert--warning {
+    // display: none;
```

Comment:
Can remove this commented line

Reply:
Cleaned up this file - this is no longer here.